player/js/elements/svgElements/SVGEffects.js

```diff
@@ -33,7 +33,7 @@ function SVGEffects(elem){
     }
     if(count){
         elem.globalData.defs.appendChild(fil);
-        elem.layerElement.setAttribute('filter','url(#'+filId+')');
+        elem.layerElement.setAttribute('filter','url(' + location.href + '#'+filId+')');
```

Comment:
Just tested an Angular page with '#' locations in url. 
location.href returns the full url, in which case the final url will have two '#'. 
Is that intended considering both scenarios (Angular and non-Angular)?